src/app/services/user.service.ts

```diff
@@ -34,4 +35,11 @@ export class UserService {
       catchError (error => throwError('Email or password was wrong'))
     )
   }
+
+  isLoggedIn (): boolean {
+    if (this.store.get('access-token') &&  this.store.get('client') && this.store.get('uid')) {
+      return true;
+    }
+    return false;
```

Comment:
Use a ternary operator:

```suggestion
 return this.store.get('access-token') &&  this.store.get('client') && this.store.get('uid')
```